src/SmartComponents/SubscriptionsUtilized/SubscriptionsUtilizedCard.js

```diff
@@ -85,35 +95,68 @@ const SubscriptionsUtilizedCard = ({ subscriptionsUtilizedProductOne, subscripti
-                    <ProgressTemplate
-                        title={ intl.formatMessage(messages.subscriptionsUtilizedProductOneTitle) }
-                        value={ productOne.percentage }
-                        label={ `${productOne.percentage}%` }
-                        variant={ (productOne.percentage <= 100 && 'info') || (productOne.percentage > 100 && 'danger') }
-                    />
-                </Tooltip>
-            ) : (<Loading key="productOneLoad" />))
-    ];
+    const charts = [];
+    const generateCharts = ({ fetchStatus, percentage, title, tooltip }) => {
+        switch (fetchStatus) {
+            case 'pending':
+                charts.push(<Loading key={ `su-loading-${title}` } />);
+                break;
+            case 'fulfilled':
+                if (percentage !== undefined && percentage !== null) {
+                    charts.push(<Tooltip key={ `su-tooltip-${title}` } content={ tooltip } position={ TooltipPosition.top } distance={ -30 }>
+                        <ProgressTemplate
+                            title={ title }
+                            value={ percentage }
+                            label={ `${percentage}%` }
+                            variant={ (percentage > 100 && 'danger') || 'info' }
+                        />
+                    </Tooltip>);
+                }
+
+                break;
+        }
+    };
+
+    generateCharts({
+        fetchStatus: subscriptionsUtilizedProductTwoFetchStatus,
+        percentage: productTwo.percentage,
+        title: intl.formatMessage(messages.subscriptionsUtilizedProductTwoTitle),
+        tooltip: productTwoTooltip
+    });
+
+    generateCharts({
+        fetchStatus: subscriptionsUtilizedProductOneFetchStatus,
+        percentage: productOne.percentage,
+        title: intl.formatMessage(messages.subscriptionsUtilizedProductOneTitle),
+        tooltip: productOneTooltip
+    });
+
+    if (productOne.percentage > productTwo.percentage && productOne.percentage > 100) {
+        charts.reverse();
+    }
 
     return <TemplateCard appName='SubscriptionsUtilized'>
         <TemplateCardHeader subtitle={ intl.formatMessage(messages.subscriptionsUtilizedTitle) }/>
         <TemplateCardBody>
-            {(productOne.percentage > productTwo.percentage && productOne.percentage > 100) ? charts.reverse() : charts}
+            {
+                (productOptIn && <EmptyState className="ins-c-subscriptions-utilized__empty-state" variant={ EmptyStateVariant.full }>
+                    <EmptyStateBody>
+                        {intl.formatMessage(messages.subscriptionsUtilizedLearnMore)}
+                    </EmptyStateBody>
+                    <EmptyStateSecondaryActions>
+                        <Button
+                            className="ins-c-subscriptions-utilized__app-link"
+                            variant="link"
+                            href="/subscriptions"
```

Comment:
Was unsure of this path implementation mostly around the `beta` vs `stable` prefix, or if it even matters since Subs Watch is meant to move to the stable environments?

Reply:
just so ah `./subscriptions` n it'll go to the place yah think it ought to no mater the env